src/lib/segmenta.ts

```diff
@@ -388,49 +370,57 @@ export class Segmenta {
             }
             const segmentName = this._generateSegmentNameFor(id);
             if (segmentName !== lastSegmentName) {
-                cmds.push(`${baseKey}/${segmentName}`);
+                cmds.push(`${ baseKey }/${ segmentName }`);
                 lastSegmentName = segmentName;
             }
             const offset = id % this._bucketSize;
             cmds.push(offset, val);
             commands++;
         }
-        debug(`setting bits with ${commands} commands`);
+        debug(`setting bits with ${ commands } commands`);
         await (this._redis as any).setbits(cmds);
     }
 
     private async _ensureSegmentExists(segment: string) {
         const index = await this._fetchIndex(segment);
         if (index.length === 0) {
-            await this._tryPut(segment, [{del: 0}]);
+            await this._tryPut(segment, [{ del: 0 }]);
         }
     }
 
     private async _fetchIndex(segment: string): Promise<string[]> {
-        const indexKey = `${this._dataKeyForSegment(segment)}/index`;
+        const indexKey = `${ this._dataKeyForSegment(segment) }/index`;
         return await this._redis.sunion(indexKey);
     }
 
     private async _getSegmentKeys(...baseKeys: string[]): Promise<string[]> {
-        return await this._redis.sunion(...baseKeys.map(bk => `${bk}/index`)) as string[];
+        return await this._redis.sunion(...baseKeys.map(bk => `${ bk }/index`)) as string[];
     }
 
     private _generateSegmentNameFor(id: number): string {
         const
             lower = Math.floor(id / this._bucketSize) * this._bucketSize,
             upper = (lower + this._bucketSize - 1);
-        return `${lower}-${upper}`;
+        return `${ lower }-${ upper }`;
     }
 
     private _dataKeyForSegment(segment: string): string {
         return this._keyGenerator.dataKeyFor(segment);
     }
 }
 
+function expectsPaging(options: ISanitizedQueryOptions): boolean {
+    return options.skip !== undefined ||
```

Comment:
Using `!==` here excludes `null`. Is that intentional?

Reply:
It was, because the typings in `segmenta` don't allow for null. However, I recognise that this may be called from outside, non-typescript code, so I've added sanity guarding to flip these incoming props to `undefined` if they are null. Making the property nullable means I have to do "if not undefined or null" checks in a bunch of places -- this function was just extracted out of the place it was being used because that place was becoming a bit unwieldy.